3rdparty/python/user_reqs.lock

```diff
@@ -4,7 +4,7 @@
 //
 // --- BEGIN PANTS LOCKFILE METADATA: DO NOT EDIT OR REMOVE ---
 // {
-//   "version": 3,
+//   "version": 2,
```

Comment:
This is the result of `./pants generate-lockfiles --resolve=python-default` on `main`. I'm not sure why that results in a downgrade of the version, but it does result in fixed `main` afaict.

Reply:
Because I'm actively changing what v3 looks like (to include `indexes` and `find_links`), so v3 is not stable yet